packages/web-pkg/src/composables/appDefaults/useAppFileHandling.ts

```diff
@@ -1,10 +1,12 @@
 import { unref } from '@vue/composition-api'
 import qs from 'qs'
 
-import { Resource } from 'files/src/helpers/resource'
+import { Resource } from 'web-client'
 import { MaybeRef } from '../../utils'
 import { ClientService } from '../../services'
 import { DavProperties } from '../../constants'
+import { buildResource } from 'files/src/helpers/resources'
```

Comment:
This should not stay in `files`, but where do we want this and a bunch of other helpers? in `web-pkg` or `web-client`?
Let's discuss tomorrow...